src/components/navigation/Nav.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import { v4 as uuid } from 'uuid';
+import logo from '../../assets/colored-logo.png';
+import routLinks from './NavRoutes';
+import SearchBar from './SearchBar';
+import styles from './navigation.module.css';
+
+export default function Nav() {
```

Comment:
- [ ] [OPTIONAL] In order to have a better UI, let's ensure the items in the nav have a yellow color as you have it in the presentation video.
![Screenshot from 2022-06-04 20-47-23](https://user-images.githubusercontent.com/58692590/172023874-5060fd8a-2fb2-4044-a2a9-6033f569ca2a.png)